### PR TITLE
Readme dependencies

### PR DESCRIPTION
**Pull request by zaeph**
Hello,

I've made some commits on top of the modifications in your original PR on `ranger/ranger`.

HTH.

### CODE REVIEW

**Review by toonn — CHANGES_REQUESTED** [review]
Two things.

File: README.md [hunk 1]
```diff
@@ -73,25 +73,27 @@ Dependencies
...
+### Optional dependencies
 
-* The `file` program for determining file types
+For general usage:
+
+* `file` for determining file types
 * `chardet` (Python package) for improved encoding detection of text files
 * `sudo` to use the "run as root" feature
-* `python-bidi` for correct display of RTL file names (Hebrew, Arabic)
+* `python-bidi` for display right-to-left file names correctly (Hebrew, Arabic)
```
Comment (toonn) [inline README.md]: "for display of" or "to display"

**Review by toonn — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -73,25 +73,27 @@ Dependencies
...
 
-Optional, for enhanced file previews (with `scope.sh`):
+More options for file previews (with `scope.sh`):
```
Comment (toonn) [inline README.md]: I kinda like the "enhanced."

**Review by toonn — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (zaeph) [reply README.md]: Reverted it.

**Review by toonn — APPROVED** [review]

**Pull request merged**